src/site/_includes/partials/subscribe.njk

```diff
@@ -0,0 +1,66 @@
+<web-subscribe id="subscribe">
+  <div class="w-subscribe">
+    <div>
+      <h1>Become a better web developer.</h1>
+      <p id="w-subscribe__message">Stay up to date with the latest about web platform.</p>
+    </div>
+    <form class="w-subscribe__form" action="{{ site.subscribeForm }}" method="post">
+      <div class="w-subscribe__fields">
+        <div class="w-subscribe__field">
+          <input class="w-subscribe__input" id="sub-firstname" name="FirstName" placeholder="First Name" required
```

Comment:
Lighthouse will grump at us because technically these elements aren't labeled for accessibility. You can create visually hidden labels (https://css-tricks.com/small-tweaks-can-make-huge-impact-websites-accessibility/#article-header-id-2) and add those to these elements.

```
<input class="w-subscribe__input" id="sub-firstname">
<label class="visually-hidden" for="sub-firstname">First name</label>
```

I'd suggest creating a new file for the visually-hidden helper class in `styles/tools/_helpers.scss`.